### ISSUE TITLE
Implement some degree of user interaction with hand gestures 

### ISSUE THREAD

**Issue by andreamazza89** [andreamazza89/smrt-with-intel-edison#3]
Maybe discrete commands, like 'next widget' associated to a movement rather than hand-tracking, unless there are existing libraries that allow this, although network speed might affect it as we have the sensor on the intel and the front end being affected on the pi...

**Comment by vannio** [comment]
@andreamazza89 I still think the Grove gesture sensor is one to consider. It recognises left/right/up/down gestures if I recall, which would be good for navigating through rather than hand-tracking as you say.

**Comment by andreamazza89** [comment]
I completely agree, so I must have not been clear :) but yes, what I am saying is we can catch discrete commands like left/right to navigate the widgets ets

**Comment by vannio** [comment]
Haha, how did I miss the 🦀?!

**Comment by andreamazza89** [comment]
Might need to use websockets to trigger events on the front end (sensor will be sitting in the backend)

🦀

**Comment by vannio** [comment]
Websockets ❤️
